src/theme/icons.tsx

```diff
@@ -619,6 +619,27 @@ const icons = {
       <path d="M20.9035 4.1178C20.9484 3.80364 20.8429 3.48648 20.6185 3.26207C20.4143 3.0661 20.133 2.9726 19.8525 3.00701C14.0847 3.7079 10.7815 6.29143 8.8936 8.83403L8.01095 8.7884C7.3684 8.75549 6.74233 8.99559 6.28754 9.45112L4.90897 10.8297C4.87531 10.8634 4.86035 10.912 4.86933 10.9584C4.87905 11.0055 4.91047 11.0444 4.9546 11.0631L7.1529 11.9749C6.90456 12.6002 6.70709 13.2443 6.56123 13.9011C6.50962 14.1375 6.58143 14.3843 6.75272 14.5563L9.361 17.1564C9.53229 17.3277 9.77914 17.3995 10.0155 17.3479C10.6723 17.202 11.3163 17.0045 11.9423 16.7562L12.8541 18.9545H12.8533C12.872 18.9986 12.9109 19.0301 12.9581 19.0398C13.0052 19.0488 13.0531 19.0338 13.0867 19.0001L14.4653 17.6216H14.466C14.9208 17.1668 15.1609 16.5407 15.128 15.8982L15.0824 15.0155C17.6196 13.1485 20.1913 9.85123 20.9033 4.11785L20.9035 4.1178ZM13.6269 12.0363C13.2738 12.3893 12.7427 12.4948 12.2812 12.304C11.8197 12.1126 11.5183 11.6622 11.5183 11.1626C11.5183 10.6629 11.8197 10.2126 12.2812 10.0219C12.7428 9.8304 13.2738 9.93662 13.6269 10.2897C14.1086 10.7714 14.1093 11.5515 13.6291 12.034L13.6269 12.0363ZM14.3928 9.53197V9.53122C13.9276 9.0667 13.7885 8.36733 14.0398 7.7592C14.2911 7.15181 14.8843 6.75537 15.5418 6.75537C16.2 6.75537 16.7924 7.15181 17.0445 7.7592C17.2959 8.36733 17.156 9.06671 16.6907 9.53122C16.3855 9.83716 15.9711 10.0092 15.5388 10.0085C15.1065 10.0085 14.692 9.83566 14.3876 9.52973L14.3928 9.53197ZM4.46987 17.6276C4.0839 17.4929 3.67025 17.46 3.26783 17.5318C3.19827 17.5438 3.12646 17.5221 3.07634 17.472C3.00977 17.4077 2.99182 17.3074 3.02997 17.2236C3.44736 16.2991 4.59629 14.4396 6.65486 15.9408V15.9401C6.67655 15.9603 6.68927 15.9879 6.68927 16.0171C6.68927 16.0463 6.67655 16.074 6.65486 16.0934C6.09161 16.5355 5.77296 17.2199 5.79689 17.9357C5.79988 18.021 5.8687 18.0898 5.95397 18.0928C6.66532 18.1212 7.3475 17.8108 7.79258 17.2558C7.81202 17.2326 7.8412 17.2184 7.87186 17.2184C7.90253 17.2184 7.93171 17.2326 7.95115 17.2558C8.23839 17.5984 9.02753 18.7114 8.15386 19.7871L8.15461 19.7878C7.77014 20.2471 7.21961 20.5366 6.6227 20.592C5.77072 20.6855 4.20438 20.954 3.55887 21.6908C3.50651 21.7551 3.42348 21.7843 3.34269 21.7671C3.26191 21.7506 3.19758 21.69 3.17589 21.6107C2.95748 20.8365 2.61488 19.1131 4.46994 17.6276L4.46987 17.6276Z" />
     ),
   },
+  'comment-filled': {
+    path: (
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M5 3C3.89543 3 3 3.89543 3 5V15C3 16.1046 3.89543 17 5 17H6.00001L6 21L11.9813 17H19C20.1046 17 21 16.1046 21 15V5C21 3.89543 20.1046 3 19 3H5Z"
+      />
+    ),
+  },
+  'markdown-icon': {
```

Comment:
we shouldn't include the word `icon` in the icon 😉 
```suggestion
  'markdown': {
```